### PR TITLE
move 1.x fetch vertice cases to tck

### PR DESCRIPTION
**Pull request by jude-zhu**

### CODE REVIEW

**Comment by jude-zhu** [comment]
from https://github.com/vesoft-inc/nebula/blob/master/src/graph/test/FetchVerticesTest.cpp

File: tests/tck/features/fetch/FetchVertices.feature [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    Then the result should be, in any order, with relax comparision:
+      | VertexID |
+
+  Scenario: [21] Fetch Vertices
+    When executing query:
+      """
+      FETCH PROP ON * hash('Boris Diaw')
+      """
+    Then the result should be, in any order, with relax comparision:
+      | -7391649757245641883 | "Boris Diaw" | 36 |
```
Comment (yixinglu) [inline tests/tck/features/fetch/FetchVertices.feature:185]: column names

File: tests/tck/features/fetch/FetchVertices.feature [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    Then the result should be, in any order, with relax comparision:
+      | VertexID             | player.name  | player.name  |
+      | -7391649757245641883 | "Boris Diaw" | "Boris Diaw" |
+
+  Scenario: [30] Fetch Vertices
+    When executing query:
+      """
+      GO FROM hash('Boris Diaw') over like YIELD like._dst as id, like._dst as id | FETCH PROP ON player $-.id YIELD player.name, player.age
+      """
+    Then a ExecutionError should be raised at runtime.
```
Comment (yixinglu) [inline tests/tck/features/fetch/FetchVertices.feature:267]: runtime:

**Comment by Shylock-Hg** [comment]
Create a new folder in `features` such as `IntVid`.

**Comment by Shylock-Hg** [comment]
FetechVertices.IntVid.feature is more clear.

**Review by yixinglu — APPROVED** [review]

**Pull request closed without merging**